src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,256 @@
+trigger:
+- master
+- main
+- release
+
+# trigger this pipeline if there's a PR to any of these branches
+pr:
+- master
+- main
+- staging
+- releases/*
+
+parameters:
+- name: Wasm
+  type: string
```

Comment:
Should these be boolean values?

Reply:
better question... why the heck are we even including these variables to conditionally build a platform... this makes no sense

Reply:
removed this on both

Reply:
Do keep in mind @arqueror we do actually need conditions using the Template Symbols that already exist to control whether or not to include the jobs for each of the various platforms. You might need to do some testing to determine what it will accept for the conditions... I don't know if yml files are actually supported out of the box or we need to define how the template processor should handle it... I would start with:
```yml
#if (ios)
job: Build iOS
....
#endif
```

If that doesn't work ping me and I'll work with you to get something working.